src/main.rs

```diff
@@ -215,7 +205,6 @@ async fn main() -> eyre::Result<()> {
 
         let tx = String::from(packages[0]); // tx_hash
         println!("get log{} {}", sdk_config.rpc_url, tx);
-        get_logs(tx, &http_provider).await;
```

Comment:
if get _logs is not needed then also remove the cmd line option section

Reply:
yeah, i'm not sure what that function was doing, just getting all logs for a given contract but don't think we need that we just need to extract logs from a receipt in the batch execute flow i think